ext/reflection/tests/bug72661.phpt

```diff
@@ -7,4 +7,4 @@ function test(iterable $arg) { }
 var_dump((string)(new ReflectionParameter("test", 0))->getType());
 ?>
 --EXPECT--
-string(8) "iterable"
+string(17) "Traversable|array"
```

Comment:
I'm confused, didn't you intend to preserve this case for BC, after discussion on https://externals.io/message/117577 (option 2)?

Reply:
I think we had a discussion with @iluuu1994 about possibly support for type aliases and how this would mess it up. I've reintroduced it for NamedTypes and hopefully made a more robust test to check that it does behave as intended.